buildSrc/src/main/groovy/org/elasticsearch/gradle/BuildPlugin.groovy

```diff
@@ -83,6 +83,19 @@ class BuildPlugin implements Plugin<Project> {
 
     /** Adds additional manifest info to jars */
     static void configureJarManifest(Project project) {
+        project.scminfo {
+            def dotgit = "$project.rootProject.projectDir/.git/"
```

Comment:
You could also calulate the hash once and store as an ext prop on the rootProject. See for example above how we only execute globalBuildInfo once.